Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.yml

```diff
@@ -585,7 +604,95 @@ script:
     - contextPath: Resilient.Incidents.Tasks.Status
       description: Task status (Open/Closed)
       type: string
-  dockerimage: demisto/resilient:1.0.0.18173
+  - arguments:
+    - default: false
+      description: The id of the incident.
+      isArray: false
+      name: incident-id
+      required: true
+      secret: false
+    - default: false
+      description: The text of the note.
+      isArray: false
+      name: comment
```

Comment:
Then why is the argument name "comment"? Was this decided by @Arsenikr ?
I think a more suitable name would be "note", @Arsenikr what do you think?